watertap/core/zero_order_costing.py

```diff
@@ -25,6 +25,7 @@
     FlowsheetCostingBlockData, register_idaes_currency_units)
 
 from watertap.core.zero_order_base import ZeroOrderBase
+from watertap.costing.watertap_costing_package import cost_membrane
```

Comment:
Can you clarify where do you use this `cost_membrane`?

Reply:
Originally, I was just going to use this method from `watertap_costing_package` but then wrote the function myself and forgot to delete this import.